### PR TITLE
Fixes #18098 - remove doubled export

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/controllers/job_templates_controller.rb [hunk 1]
```diff
@@ -41,11 +41,6 @@ def import
     end
   end
 
-  def export
```
Comment (iNecas) [inline app/controllers/job_templates_controller.rb]: should be removed from `routes.rb`: this is the reason why the tests are failling on permission check

**Comment by ares** [comment]
Good catch, I removed it for UI, hopefully tests will be green now.

**Comment by ares** [comment]
@iNecas mind to take another look?

**Comment by iNecas** [comment]
Jenkins is happy => me is happy

**Pull request merged**